apps/gp2-server/test/controllers/working-group.controller.test.ts

```diff
@@ -137,4 +133,23 @@ describe('Working Group controller', () => {
       });
     });
   });
+
+  describe('update', () => {
+    beforeEach(jest.resetAllMocks);
+
+    test('Should return the newly updated user', async () => {
+      const mockResponse = getWorkingGroupDataObject();
+      workingGroupDataProviderMock.fetchById.mockResolvedValue(mockResponse);
+      const result = await workingGroupController.update(
+        '7',
+        { resources: [] },
+        '11',
+      );
+
+      expect(result).toEqual(getWorkingGroupResponse());
+      expect(workingGroupDataProviderMock.update).toHaveBeenCalledWith('7', {
+        resources: [],
```

Comment:
probably best to test with actual data instead?

Reply:
an empty array is valid too. I've updated it.